<?xml version="1.0" encoding="UTF-8"?>
<!DOCTYPE suite SYSTEM "https://testng.org/testng-1.0.dtd">
<suite name="Failed suite [Default suite]" guice-stage="DEVELOPMENT">
  <test thread-count="5" name="Default test(failed)">
    <classes>
      <class name="exercise.VendingMachineTest">
        <methods>
          <include name="testAddCoin_invalidCoin"/>
          <include name="testReset_bySupplier_wrongPasscode"/>
          <include name="testSelectProduct_invalidProduct_shouldThrowException"/>
          <include name="testVendItemAndrefund_insufficientAmountProvided"/>
          <include name="testVendItemAndrefund_sufficientAmountProvided"/>
          <include name="testReset_bySupplier_correctPasscode"/>
          <include name="testVendItemAndrefund_ExtraAmountProvided"/>
          <include name="testSelectProduct_unavailableProduct_shouldThrowException"/>
          <include name="testCancelAndRefund_someAmountProvided"/>
        </methods>
      </class> <!-- exercise.VendingMachineTest -->
    </classes>
  </test> <!-- Default test(failed) -->
</suite> <!-- Failed suite [Default suite] -->
